app/controllers/api/content_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::ContentController < Api::BaseController
+  before_action :validate_params!
+  def index
+    @content = Reports::Content.retrieve(from: params[:from], to: params[:to],
+    organisation: params[:organisation])
+    render json: { results: @content }.to_json
+  end
+
+  def api_request
+    @api_request ||= Api::Request.new(params.permit(:from, :to, :metric, :base_path, :organisation, :format, metrics: []),
```

Comment:
it would be nice to extract ```params.permit(:from, :to, :metric, :base_path, :organisation, :format, metrics: []``` into a method like ```request_parameters```